.travis.yml

```diff
@@ -53,3 +53,7 @@ jobs:
       script:
         - PKG_OS=linux make packages
         - DOCKER_PUSH_LATEST=true make docker-push
+    - name: 'Deploy to staging'
+      stage: release
```

Comment:
I think it will be nicer to add an install phase that does a `make install-helm`